implementations.yaml

```diff
@@ -35,11 +35,11 @@ kems:
 
       This is not relevant for the 'normal' api methods.
     schemes:
-      - name: frodokem640shake
```

Comment:
Frodo also got updated in PQClean so it should get a version bump (patch version level) — these fixes should not have affected compatibility.

https://github.com/PQClean/PQClean/pull/367

Reply:
(Github wouldn't let me comment in the right place with a suggestion)

Reply:
Update note in changelog added.